### ISSUE TITLE
Moving generated code into libcraft?

### ISSUE THREAD

**Issue by Miro-Andrin** [feather-rs/feather#459]
I was looking at adding recipes to libcraft  #429 , but noticed that Item and ItemStack are in feather, under the generated folder.  
Any reason why we should not move all the generated code from feather to libcraft? 

## Description of change.
Move feather/feather/generated/* into feather/libcraft/generated 
And make the generated structs repr-c such that we can use them in quill.

**Comment by koskja** [comment]
I agree, maybe we could also move the feather-generated and the dapapack crate there.
